modin/core/dataframe/pandas/partitioning/partition_manager.py

```diff
@@ -1148,6 +1148,84 @@ def _apply_func_to_list_of_partitions(cls, func, partitions, **kwargs):
         preprocessed_func = cls.preprocess_func(func)
         return [obj.apply(preprocessed_func, **kwargs) for obj in partitions]
 
+    @classmethod
+    def convert_partitions_to_pandas_in_remote_func(cls, partitions):
```

Comment:
```suggestion
    def combine(cls, partitions):
```
or to_single_partition